config/functions.config.staging.js

```diff
@@ -1,15 +1,13 @@
 var config = {
   client: {
-    url: "http://localhost:3000/",
+    url: "https://staging-permissions.covidwatch.org/",
```

Comment:
Should we also update this field in the prod file?

Reply:
Given this is the file `config/functions.config.staging.js` shouldn't this be our staging URL? permission-portal.covidwatch.org.  

Accordingly, shouldn't `config/functions.config.prod.js` have the prod URL: portal.covidwatch.org?